index.js

```diff
@@ -100,11 +101,20 @@ function addExceptionsToUnprotected (exceptions) {
 }
 
 const listData = JSON.parse(fs.readFileSync(`${LISTS_DIR}/${unprotectedListName}`))
-defaultConfig.unprotectedTemporary = listData.exceptions
-
-addExceptionsToUnprotected(defaultConfig.unprotectedTemporary)
+addExceptionsToUnprotected(listData.exceptions)
 addExceptionsToUnprotected(defaultConfig.features.contentBlocking.exceptions)
 
+const notExcluded = ['adClickAttribution', 'appTrackerProtection', 'autofill', 'duckPlayer', 'incontextSignup',
+    'incrementalRolloutTest', 'networkProtection', 'newTabContinueSetUp', 'voiceSearch', 'windowsPermissionUsage',
+    'windowsWaitlist', 'windowsDownloadLink']
+for (const exception of listData.exceptions) {
+    for (const key of Object.keys(defaultConfig.features)) {
+        if (!(notExcluded.includes(key))) {
+            defaultConfig.features[key].exceptions = defaultConfig.features[key].exceptions.concat(exception)
```

Comment:
`concat` can accept an array, we can remove the `const exception of listData.exceptions` and make this line:
```suggestion
            defaultConfig.features[key].exceptions = defaultConfig.features[key].exceptions.concat(listData.exceptions)
```